com.woltlab.wcf/option.xml

```diff
@@ -719,11 +719,6 @@ block:wcf.acp.option.blacklist_sfs_action.block]]></selectoptions>
 				<optiontype>boolean</optiontype>
 				<defaultvalue>1</defaultvalue>
 			</option>
-			<option name="search_use_captcha">
-				<categoryname>security.antispam.captcha</categoryname>
-				<optiontype>boolean</optiontype>
-				<defaultvalue>0</defaultvalue>
-			</option>
```

Comment:
With the removal of the captcha (which I consider a good thing), we probably should add support for the flood control with a reasonable limit to prevent the most obvious cases of abuse / DoS attacks. Something like the following should affect any legitimate uses:

- at most 20 searches within the last 60 seconds, but not more than
- 300 searches within the last 12 hours.